flow/embedded_views.h

```diff
@@ -42,8 +56,88 @@ class ExternalViewEmbedder {
   virtual ~ExternalViewEmbedder() = default;
 
   FML_DISALLOW_COPY_AND_ASSIGN(ExternalViewEmbedder);
+
+#pragma mark - transforms
+  std::shared_ptr<FlutterEmbededViewTransformStack> transformStack;
```

Comment:
This is the state of an ongoing paint traversal, I do not think it belongs in ExternalViewEmbedder(e.g think of an hypothetical world where we'll be painting multiple parts of the tree in parallel). I'd keep this in PaintContext.

Reply:
Sure. Do you think all the stack related classes belong to the same file of paintContext?

Reply:
Done